pkg/asset/k8s.go

```diff
@@ -163,7 +163,7 @@ func secretFromAssets(name, namespace string, assetNames []string, assets Assets
 	})
 }
 
-func mustCreateAssetFromTemplate(name string, template []byte, data interface{}) Asset {
+func MustCreateAssetFromTemplate(name string, template []byte, data interface{}) Asset {
```

Comment:
Can you elaborate on why you need to make this a public API?

Currently in bootkube recovery tool we simply return an Assets array, and then the caller invokes `Assets.WriteFiles()` to write them out to disk.